src/ert/simulator/simulation_context.py

```diff
@@ -72,7 +73,7 @@ def _run_forward_model(
             )
         ]
 
-    job_queue.execute_queue(queue_evaluators)
+    asyncio.run(job_queue.execute(evaluators=queue_evaluators))
```

Comment:
`run_until_complete`? calling `asyncio.run` always creates a new event loop actually, but maybe that's what we want here?

Reply:
asyncio.run() is the Python 3.7 way of doing asyncio.get_event_loop().run_until_complete() (which is from python 3.4)

Reply:
yes, we do need a new event loop here, because we are coming from a totally sync context (everest)